metrics/metrics_collector.go

```diff
@@ -137,18 +154,37 @@ func (mc *metricCollector) registerHistogram(count int) {
 func (mc *metricCollector) registerSummary(count int) {
 	for idx := 0; idx < count; idx++ {
 		namespace := "test"
-		summary := prometheus.NewSummary(
+		summary := prometheus.NewSummaryVec(
 			prometheus.SummaryOpts{
-				Namespace: namespace,
-				Name:      fmt.Sprintf("summary%v", idx),
-				Help:      "This is my summary",
-				Objectives: map[float64]float64{
-					0.1:  0.5,
-					0.5:  0.5,
-					0.99: 0.5,
-				},
-			})
+				Namespace:  namespace,
+				Name:       fmt.Sprintf("summary%v", idx),
+				Help:       "This is my summary",
+				Objectives: map[float64]float64{0.5: 0.05, 0.9: 0.01, 0.99: 0.001},
```

Comment:
What does this change will do?

Reply:
I have reversed this change. I was trying different quantile ranges, so left them there. I have reversed it now.